src/nodes.coffee

```diff
@@ -874,6 +875,12 @@ exports.StringLiteral = class StringLiteral extends Literal
     unquoted = unquoted.replace /\\n/g, '\n' if csx
     unquoted
 
+  astProperties: ->
```

Comment:
There will be more to string/heredoc ASTs but this gets a basic version in place